perses/tests/test_repex.py

```diff
@@ -79,12 +82,17 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
             f_ij, df_ij = analyzer.get_free_energy()
             data[f"{wt_name}-{mutant_name}"] = {'free_energy': f_ij[0, -1], 'error': df_ij[0, -1]}
 
-        DDG = abs(data['ala-thr']['free_energy'] - data['thr-ala']['free_energy'] * -1)
-        dDDG = np.sqrt(data['ala-thr']['error'] ** 2 + data['thr-ala']['error'] ** 2)
-        assert DDG < 6 * dDDG, f"DDG ({DDG}) is greater than 6 * dDDG ({6  * dDDG})"
+        forward_fe = data['ala-thr']['free_energy']
+        reverse_fe = data['thr-ala']['free_energy']  # This should have the inverse sign of the forward fe
+        # they should add up to close to zero
+        forward_reverse_diff = abs(forward_fe + reverse_fe)
```

Comment:
should we call this `forward_reverse_sum` since its a sum and not a difference? same with the line below it, maybe we should call it `forward_reverse_sum_error`

Reply:
Solved by changes in https://github.com/choderalab/perses/pull/1065/commits/629b20e4de1ab9fc6fdab4af6bba830e6d4e4aed